pkgs/by-name/zl/zluta/package.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, lib, fetchFromGitHub, rocmPackages, python3, cargo, rustc, cmake, clang, zlib, libxml2, libedit, pkgs }:
+
+pkgs.rustPlatform.buildRustPackage rec {
+  pname = "zluda";
+  version = "3";
+
+  src = fetchFromGitHub {
+    owner = "vosen";
+    repo = "ZLUDA";
+    rev = "v${version}";
+    hash = "sha256-lykM18Ml1eeLMj/y6uPk34QOeh7Y59i1Y0Nr118Manw=";
+    fetchSubmodules = true;
```

Comment:
Are there many of those? Is it feasible to do without submodules?

Reply:
👍🏻 it's ok to start out with the submodules, maybe someone refactors this in the future